bser.c

```diff
@@ -186,6 +201,13 @@ static int bser_bytestring(const char *str, json_dump_callback_t dump, void *dat
   return 0;
 }
 
+static int bser_bytestring(const char *str, json_dump_callback_t dump,
```

Comment:
Yes. I originally wrote bser_/bunser_ versions for utf8 but they are not currently used so the compiler complained about them.